src/main/drivers/sdcard/sdmmc_sdio.h

```diff
@@ -215,7 +219,12 @@ typedef struct
 extern SD_CardInfo_t SD_CardInfo;
 extern SD_CardType_t SD_CardType;
 
-bool SD_Initialize_LL(DMA_Stream_TypeDef * dmaRef);
+#ifdef AT32F43x
+// TODO:sd卡还不支持，待处理
```

Comment:
Can you provide english translation of the comments?

Reply:
current the SD card is not support

Reply:
Do any of the new targets come with an sd card slot? 

Can you update the comments in the pull request to be in English. Although English  is not my native either, it is the language used in the code and documentation.

Reply:
If the hardware in question does not include sdcard slots, you may also change the comment to be

#ifdef AT32F43x
#error  SD card not implemented
void             SD_Initialize_LL            (dma_channel_type *dma);
#else
void             SD_Initialize_LL            (DMA_Stream_TypeDef *dma);
#endif

Reply:
> If the hardware in question does not include sdcard slots, you may also change the comment to be
> 
> #ifdef AT32F43x #error SD card not implemented void SD_Initialize_LL (dma_channel_type *dma); #else void SD_Initialize_LL (DMA_Stream_TypeDef *dma); #endif

ok, comment in english will be added , but could i keep the chinese version in it ?  i wants to use both